pkg/cinderapi/deployment.go

```diff
@@ -91,7 +91,7 @@ func Deployment(
 			Selector: &metav1.LabelSelector{
 				MatchLabels: labels,
 			},
-			Replicas: &instance.Spec.Replicas,
+			Replicas: instance.Spec.Replicas,
```

Comment:
This part confuses me (even the original code). According to the [StatefulSetSpec](https://pkg.go.dev/k8s.io/kubernetes/pkg/apis/apps#StatefulSetSpec), the Replicas field is an int32 (not a pointer), so I don't know how we can mix pointers and pointer references.

Reply:
For whatever reason, that documentation is not accurate.  If you look at the code in an interactive IDE, the imported libraries for `StatefulSet`, `Deployment`, etc, have an `*int32` as the data type for that field.

Reply:
Ah, thanks for explaining this as a doc error. You've saved me from an eternity of staring into the abyss.